amgut/lib/data_access/test/test_survey.py

```diff
@@ -230,6 +229,56 @@ def test_american_name(self):
 
 
 class TestSurvey(TestCase):
+    def insert_data(self):
+        """ Prepare tests by inserting some data into the DB.
+
+        Returns
+        -------
+        survey object, survey_id, notes_test and consent of the newly created
```

Comment:
no. I specified types as follows:
`survey object, survey_id : str, notes_test : str and consent : dict of the newly created survey.`